tests/search/schemachanges/schemachanges_need_refeed_reconfig.rb

```diff
@@ -39,9 +39,10 @@ def test_need_refeed_reconfig
     assert_match(/Consider re-indexing document type 'test' in cluster 'search'.*\n.*Field 'f2' changed: add index aspect/, redeploy_output)
 
     # Wait up to 2 minutes for reindexing to be ready
+    reindexingUrl = "https://#{vespa.configservers["0"].name}:#{vespa.configservers["0"].ports[1]}/application/v2/tenant/#{@tenant_name}/application/#{@application_name}/environment/prod/region/default/instance/default/reindexing"
```

Comment:
Please use snake case for ruby code: reindexing_url

Reply:
Of course!